src/app/cli/src/coda_worker.ml

```diff
@@ -272,6 +272,7 @@ module Make (Kernel : Kernel_intf) = struct
         in
         let coda_prove_receipt (proving_receipt, resulting_receipt) =
           Run.prove_receipt coda ~proving_receipt ~resulting_receipt
+          |> Deferred.Or_error.ok_exn
```

Comment:
when can this fail? It would be nice to print a clean error message when it does fail instead of showing a stack trace to the user